ci/Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM ubuntu:bionic
 
-RUN apt-get update && apt-get install openjdk-8-jdk git python2.7 python-setuptools python-six python-yaml sudo maven -y
+RUN apt-get update && apt-get install openjdk-8-jdk git python2.7 python-setuptools python-six python-yaml sudo maven curl -y
```

Comment:
Just wondering: do we manually rebuild the docker image after changing this file?

Reply:
Yes, I don't plan on it changing very much once we get it locked in.